Source/Cvent.SchemaToPoco.Core/JsonSchemaToCodeUnit.cs

```diff
@@ -197,7 +197,7 @@ public static CodeMemberField CreateProperty(string name, CodeTypeReference type
                 
             };
 
-            field.Name += " { get; set; }";
+            field.Name += " { get; set; } //";
```

Comment:
This is to comment out a semicolon that gets added erroneously, right? I would definitely want to see this commented.